integration-tests/environments/react-native/package.json

```diff
@@ -53,6 +53,8 @@
     "realm-app-importer": "*"
   },
   "localDependencies": {
+    "@realm.io/common": "../../../packages/realm-common",
+    "realm-network-transport": "../../../packages/realm-network-transport",
```

Comment:
This shouldn't be needed (or they should be removed again) after https://github.com/realm/realm-js/pull/4054 merge (depending on what happens first 😅 )

Reply:
👍  I am just reviewing that one now so if we merge that first I can update this after!